docs: add changelogs (#233)
qase-tms/qase-python: qase-robotframework/changelog.md

# qase-pytest 3.1.0b1

## What's new

> Implement a method which constructs the signature to each test result.
> It takes the file path, the suites, the qase ids and the parameters.

Reviewer: ```suggestion
Implemented a method that constructs a signature for each test result.
It takes the file path, suites, qase IDs, and parameters.
```

Author: fixed

# qase-robotframework@3.0.0

## What's new

> This is the first release in the 3.0.x series of the Robot Framework reporter.

Reviewer: ```suggestion
The first release in the 3.0.x series of the Robot Framework reporter.
```

# qase-robotframework@3.0.0b4

## What's new

> The logic for extracting Qase ID has been changed. The reporter will extract the Qase ID after ending test.

Reviewer: ```suggestion
The reporter will now extract the Qase ID after ending the test.
```

Author: fixed

# qase-robotframework@3.0.0b3

## What's new

Fixed the issue:

```log
AttributeError: 'Result' object has no attribute 'case'
```

> Renamed the internal package name `qaseio` to `qase`

Reviewer: ```suggestion
Renamed the internal API client package from `qaseio` to `qase`.
```

# qase-robotframework@3.0.0b2

## What's new

> Fixed an issue with unsupported step status. If the step is `skipped` then mark the step as `blocked`.

Reviewer: ```suggestion
Fixed an issue with unsupported step status. If the step is `skipped`, mark such a step as `blocked` in the Qase test run.
```

Author: fixed
